### PR TITLE
feature/EUI-8378

### PR DESCRIPTION
**Pull request by Kartheek-S**
### JIRA link (if applicable) ###
https://tools.hmcts.net/jira/browse/EUI-8378

### Change description ###
Query management- Respond query details (Function)

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[ x ] No
```

### CODE REVIEW

File: .github/workflows/npmpublish.yml [hunk 1]
```diff
@@ -20,7 +20,7 @@ jobs:
       - uses: actions/checkout@v3
       - uses: actions/setup-node@v3
         with:
-          node-version: 16
+          node-version: 14
```
Comment (connorpgpmcelroy) [inline .github/workflows/npmpublish.yml:23]: node-version has gone backwards here? Should be consistent with Release if that is where it is going

File: package.json [hunk 2]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "@hmcts/ccd-case-ui-toolkit",
-  "version": "6.10.7-case-flags-v2-welsh-language-support-additional-rework-2",
+  "version": "6.16.0-query-management-write-doc-view-v10-fix",
```
Comment (connorpgpmcelroy) [inline package.json:3]: Again, version had gone backwards - if going to Release we need to be consistent with Release

**Review by connorpgpmcelroy — CHANGES_REQUESTED** [review]
Version changes needed

**Review by DavidJayakumar — APPROVED** [review]
LGTM

**Review by DavidJayakumar — COMMENTED** [review]
Please revolve conflicts

File: .github/workflows/npmpublish.yml (hunk 1 above)
Comment (Kartheek-S) [reply .github/workflows/npmpublish.yml]: branch rebased with release

**Review by connorpgpmcelroy — APPROVED** [review]
LGTM

**Review by DavidJayakumar — APPROVED** [review]

**Comment by MateiHMCTS** [comment]
Closing PR as proposed solution is not in line with requirements.

**Pull request closed without merging**